topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1214 @@
+
+> <hands-on-title>Preprocessing data</hands-on-title>
+>
+> 2. Run {% tool [Collapse Collection Tool](toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0) %} with the following parameters:
+>    - {% icon param-collection %} *"Collection of files to collapse into single dataset"*: `Short reads`
+>    - *"Keep one header line"*: `No`
+>    - *"Prepend File name"*: `No`
+>
+> 4. {% tool [FASTQ splitter](toolshed.g2.bx.psu.edu/repos/devteam/fastq_paired_end_splitter/1.1.5+galaxy1) %} with following parameters:
+>     - {% icon param-file %} *"FASTQ reads"*: output of Collapse collection tool
+> 5. Rename the outputs generated by **FASTQ splitter** to include the information about forward and reverse if the information is not in the name
+>
+>    > <comment-title></comment-title>
+>    > While working on this project we realised that both outputs (forward and reverse) had the same name `FASTQ splitter on data X`, where X is the internal number of data in this Galaxy history, with the version of FASTQ splitter we used
+>    > Since we have run FASTQ splitter on one dataset and got 2 outputs, as a result filenames of both outputs were the same.
+>    >
+>    > Why is it not good?
+>    >
+>    > It can be mistaken by assembler (or any other tool where we want to use FASTQ splitter output files) because it will not know which file is reverse and which is forward.
+>    >
+>    > We updated the Galaxy tool to fix this issue
+>    {: .comment}
+>
+> 6. Repeat the steps for long reads
+{: .hands_on}
+
+We now have collections for both short and long reads.
+
+# Select and prepare the tools to run
+
+Based on {% cite meyer2022critical %}, {% cite Meyer2021 %} and {% cite Meyer2021_tutorial %}, we can compare tools on a set of metrics to select the one to use for an analysis but also here to run the challenge:
+
+Tool | Genome fraction (%) | Mismatches per 100 kbp | Misassemblies | NGA50 | Strain recall | Strain precision
+Gold Standard Assembly (GSA) | 76.9 | 0 | 0 | 682,777 | 54.9% (upper bound) | 100
+ABySS ({% cite jackman2017abyss %}) | Very accurate mean fraction (<1% divergence) |  | The fewest misassemblies | 100% precision. The highest strain precision (100%) for the unique genome
+Ray ({% cite Boisvert2012 %}) |  |  |  |  |  | 100% precision. The highest strain precision (100%) for the unique genome
+A-STAR | A-STAR excelled in terms of genome fraction on marine and strain madness data sets. A-STAR improved the genome fraction to 44.1% on the marine dataset. On marine common genomes, A-STAR (26.7%) achieved the highest genome fractions. For unique genome, A-STAR provided the most complete assemblies (55.3% genome fraction). A-STAR partially recovered 102 (78%) of 131 16S gold standard sequences. | More mismatches than others: 773/100 kb | More misassemblies than others | | 2nd highest: 7.5% recall  | 2nd highest: 69.4% precision
```

Comment:
Could you simply the content of the cells in this table and the ones in the detail box?